packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -25,9 +25,9 @@ import {
   edmToComplexPropertyType,
   isNullableParameter
 } from '../../../generator-utils';
-import { applyPrefixOnJsConfictParam } from '../../../name-formatting-strategies';
+import { applyPrefixOnJsConfictParam as applyPrefixOnJsConflictParam } from '../../../name-formatting-strategies';
```

Comment:
Nice that you caught the type. But could you perhaps deprecate the old method and creat a new copy with the proper name so that we remember to remove the type it in version 2.0